client/components/logout/__snapshots__/logout.test.js.snap

```diff
@@ -23,7 +23,39 @@ exports[`<Logout /> rendering should render correctly 1`] = `
             onClick={[Function]}
             to="/default/status"
           >
-            Login Again
+            Log in again
+          </Link>
+        </div>
+      </div>
+    </div>
+  </div>
+</React.Fragment>
+`;
+
+exports[`<Logout /> rendering with placeholder translation tags should render translation placeholder correctly 1`] = `
+<React.Fragment>
+  <div
+    className="container content"
+    id="logout"
+  >
+    <div
+      className="inner"
+    >
+      <div
+        className="main-column w-100"
+      >
+        <h2>
+          LOGOUT_CONTENT
```

Comment:
I added a test for rendering with placeholder tags, so that in the future if placeholder tags are changed then the snapshot must fail, and updating `po` files are required. Same reason for other occurences also.